at-quannguyen/W4/ex1.rb

```diff
@@ -0,0 +1,13 @@
+def operator
+  Proc.new
+end
+
+add = operator { |*arr| arr.inject(:+) }
+sub = operator { |*arr| arr.inject(:-) }
+mul = operator { |*arr| arr.inject(:*) }
+div = operator { |*arr| arr.inject(:/) rescue puts 'can not div by zero' }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/cd24ce24c04c57d4b75cc24b8a37d518f57e5e0e/at-quannguyen/W4/ex1.rb#L8"></span>Style/RescueModifier: Avoid using <code>rescue</code> in its modifier form. (https://github.com/rubocop-hq/ruby-style-guide#no-rescue-modifiers)</td>
    </tr>
  </tbody>
</table>